docs/source/ivr.rst

```diff
@@ -278,6 +281,9 @@ Settings
 
         .. [#] Note: no square brackets
 
+        A configuration can be set as the default configuration from the UI.
```

Comment:
Only by the UI. What with config file ? Maybe give it up, before "Settings" screenshot. And add information about default config, something like there is possibility to set config as default